shoop/core/order_creator/_modifier.py

```diff
@@ -17,9 +19,14 @@ class OrderModifier(OrderProcessor):
     @atomic
     def update_order_from_source(self, order_source, order):
         data = self.get_source_base_data(order_source)
+        data.update({
+            "modified_by": real_user_or_none(order_source.modified_by),
+            "modified_on": now()
+        })
         Order.objects.filter(pk=order.pk).update(**data)
```

Comment:
Some data should never be updated in this process, e.g. creator or ip_address. You should drop them from the data dictionary before calling the `Order.update(**data)`